converter/bijection/types.go

```diff
@@ -0,0 +1,6 @@
+package bijection
+
+type Bijection[A interface{}, B interface{}] interface {
+	ConvertAToB(a *A, b *B) error
+	ConvertBToA(b *B, a *A) error
+}
```

Comment:
This is the idea in essence - a definition of a conversion that can be done in both directions.